llvm/lib/Transforms/Utils/SCCPSolver.cpp

```diff
@@ -2125,6 +2143,21 @@ void SCCPInstVisitor::handleCallResult(CallBase &CB) {
   }
 }
 
+bool SCCPInstVisitor::isInstUnderDefined(Instruction &Inst) {
+  // For structure Type, we handle each member seperately.
+  // A structure object won't be considered as overDefined when
+  // there is at least one member can become constant.
```

Comment:
```suggestion
  // there is at least one member that can become constant.
```
Though I'd say "at least one member that is not overdefined", because constants are not the only lattice values we care about.